Traceback (most recent call last):
  File "/usr/lib/python3.9/runpy.py", line 197, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/usr/lib/python3.9/runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "/mnt/c/Users/jayson/source/repos/frappe/abkd_app/apps/frappe/frappe/utils/bench_helper.py", line 110, in <module>
    main()
  File "/mnt/c/Users/jayson/source/repos/frappe/abkd_app/apps/frappe/frappe/utils/bench_helper.py", line 18, in main
    commands = get_app_groups()
  File "/mnt/c/Users/jayson/source/repos/frappe/abkd_app/apps/frappe/frappe/utils/bench_helper.py", line 27, in get_app_groups
    for app in get_apps():
  File "/mnt/c/Users/jayson/source/repos/frappe/abkd_app/apps/frappe/frappe/utils/bench_helper.py", line 104, in get_apps
    return frappe.get_all_apps(with_internal_apps=False, sites_path=".")
  File "/mnt/c/Users/jayson/source/repos/frappe/abkd_app/apps/frappe/frappe/__init__.py", line 1257, in get_all_apps
    apps = get_file_items(os.path.join(sites_path, "apps.txt"), raise_not_found=True)
  File "/mnt/c/Users/jayson/source/repos/frappe/abkd_app/apps/frappe/frappe/__init__.py", line 1402, in get_file_items
    content = read_file(path, raise_not_found=raise_not_found)
  File "/mnt/c/Users/jayson/source/repos/frappe/abkd_app/apps/frappe/frappe/__init__.py", line 1430, in read_file
    raise IOError("{} Not Found".format(path))
OSError: b'./apps.txt' Not Found

Traceback (most recent call last):
  File "/usr/lib/python3.9/runpy.py", line 197, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/usr/lib/python3.9/runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "/mnt/c/Users/jayson/source/repos/frappe/abkd_app/apps/frappe/frappe/utils/bench_helper.py", line 110, in <module>
    main()
  File "/mnt/c/Users/jayson/source/repos/frappe/abkd_app/apps/frappe/frappe/utils/bench_helper.py", line 18, in main
    commands = get_app_groups()
  File "/mnt/c/Users/jayson/source/repos/frappe/abkd_app/apps/frappe/frappe/utils/bench_helper.py", line 27, in get_app_groups
    for app in get_apps():
  File "/mnt/c/Users/jayson/source/repos/frappe/abkd_app/apps/frappe/frappe/utils/bench_helper.py", line 104, in get_apps
    return frappe.get_all_apps(with_internal_apps=False, sites_path=".")
  File "/mnt/c/Users/jayson/source/repos/frappe/abkd_app/apps/frappe/frappe/__init__.py", line 1257, in get_all_apps
    apps = get_file_items(os.path.join(sites_path, "apps.txt"), raise_not_found=True)
  File "/mnt/c/Users/jayson/source/repos/frappe/abkd_app/apps/frappe/frappe/__init__.py", line 1402, in get_file_items
    content = read_file(path, raise_not_found=raise_not_found)
  File "/mnt/c/Users/jayson/source/repos/frappe/abkd_app/apps/frappe/frappe/__init__.py", line 1430, in read_file
    raise IOError("{} Not Found".format(path))
OSError: b'./apps.txt' Not Found

Traceback (most recent call last):
  File "/usr/lib/python3.9/runpy.py", line 197, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/usr/lib/python3.9/runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "/mnt/c/Users/jayson/source/repos/frappe/abkd_app/apps/frappe/frappe/utils/bench_helper.py", line 110, in <module>
    main()
  File "/mnt/c/Users/jayson/source/repos/frappe/abkd_app/apps/frappe/frappe/utils/bench_helper.py", line 18, in main
    commands = get_app_groups()
  File "/mnt/c/Users/jayson/source/repos/frappe/abkd_app/apps/frappe/frappe/utils/bench_helper.py", line 27, in get_app_groups
    for app in get_apps():
  File "/mnt/c/Users/jayson/source/repos/frappe/abkd_app/apps/frappe/frappe/utils/bench_helper.py", line 104, in get_apps
    return frappe.get_all_apps(with_internal_apps=False, sites_path=".")
  File "/mnt/c/Users/jayson/source/repos/frappe/abkd_app/apps/frappe/frappe/__init__.py", line 1257, in get_all_apps
    apps = get_file_items(os.path.join(sites_path, "apps.txt"), raise_not_found=True)
  File "/mnt/c/Users/jayson/source/repos/frappe/abkd_app/apps/frappe/frappe/__init__.py", line 1402, in get_file_items
    content = read_file(path, raise_not_found=raise_not_found)
  File "/mnt/c/Users/jayson/source/repos/frappe/abkd_app/apps/frappe/frappe/__init__.py", line 1430, in read_file
    raise IOError("{} Not Found".format(path))
OSError: b'./apps.txt' Not Found

